File: python/88. Merge Sorted Array/using_two_pointer.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        i=0
        for x in range(m,m+n):
            nums1[x]=nums2[i]
            i+=1
            if(i==n):
                break
        nums1.sort()

File: C++/88. Merge Sort Array/using_two_pointer.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int i=m,j=0;
        while (i!=(m+n)){
            if (nums1.size()>=i){
                nums1[i]=nums2[j];
                i++;
                j++;
            }
            else{
                nums1.push_back(nums2[j]);
                i++;
                j++;
            }
        }
        sort(nums1.begin(),nums1.end());
    }
};